config.go

```diff
@@ -114,13 +107,7 @@ func (c *Config) setDefaults() {
 	}
 }
 
-func redisClient(redisURL string) (*redis.Client, error) {
-	client := redis.NewClient(&redis.Options{
-		Addr: redisURL,
-	})
-	_, err := client.Ping().Result()
-	if err != nil {
-		return nil, err
-	}
-	return client, nil
+func redisClientDefault() *nrredis.ClusterClient {
+	options := nrredis.NewClusterModeDisabledOptions([]string{defaultRedisAddr})
+	return nrredis.NewClusterClient(options)
```

Comment:
This looks like a little change in behavior. It looks live previously we were making sure the connection was successful by pinging. Should we keep the old behavior? Or is there a good reason to remove that behavior?